javaparser-core/src/main/java/com/github/javaparser/jctree/ast/AJCAnnotation.java

```diff
@@ -0,0 +1,38 @@
+package com.github.javaparser.jctree.ast;
+
+import com.sun.tools.javac.tree.JCTree.*;
+import com.sun.tools.javac.tree.JCTree;
+import com.sun.tools.javac.util.List;
+//import com.sun.tools.javac.util.Name;
+//import com.sun.tools.javac.code.Symbol;
+//import com.sun.tools.javac.code.Scope.ImportScope;
+//import com.sun.tools.javac.code.BoundKind;
+
+public class AJCAnnotation extends JCAnnotation implements JavaParserComments {
```

Comment:
Sure, I can do both. The classes extend the JCTree AST nodes, so it preserve the comments per-node.

Do you have suggestions for the new name?